src/applications/check-in/pre-check-in/pages/Landing/index.jsx

```diff
@@ -102,15 +105,22 @@ const Index = props => {
                 }
               }
             })
-            .catch(() => {
+            .catch(error => {
               clearCurrentSession(window);
-              goToErrorPage();
+              if (error.errors?.find(err => err.status === '401')) {
```

Comment:
```suggestion
              if (error.status === 401) {
```

Reply:
What would you think of doing this? I was testing changing the mock response status locally and couldn't trigger the error and then I realized it was because this tests the response body errors rather than the http status.

Reply:
We talked this over and it seems like due to how the platform util handles API requests we don't have the status code available to us.